src/Angor/Client/Pages/View.razor

```diff
@@ -394,10 +416,15 @@
     string myProjectExplorerLink;
     string projectExplorerLink;
 
-    private string NostrSecKey { get; set; } = string.Empty;
+    private string NostrNsecSecKey { get; set; } = string.Empty;
+    private string NostrHexSecKey { get; set; } = string.Empty;
+
+    private string Npub => NostrConversionHelper.ConvertHexToNpub(project.ProjectInfo.NostrPubKey)!;
```

Comment:
is this object always guaranteed to be not null?

Reply:
removed !

Reply:
because I used another var in the html codes